src/components/MatPass.tsx

```
@@ -0,0 +1,478 @@
+    detail: { ...item, quantity: 1 },
+    bubbles: true 
+  });
+  document.dispatchEvent(event);
+};
+
+interface MatPassProps {
+  parentTab?: string;
+}
+
+const getPackageNumber = (name: string): number => {
+  const number = parseInt(name.split(' ')[0] || '1', 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)